config/jail.conf

```diff
@@ -344,7 +344,7 @@ logpath = %(lighttpd_error_log)s
 [roundcube-auth]
 
 port     = http,https
-logpath  = /var/log/roundcube/userlogins
+logpath  = /var/log/roundcubemail/errors
```

Comment:
I checked out the Debian package and it does seem to define /var/log/roundcube; so I can understand adding that to config/paths-*. However, the file name was not blindly changed. (Actually I posed the path question to the mailing list specifically regarding other platforms but received no feedback - so that was not done blindly either.)

IMHO, default fail2ban should aim to work with default roundcube. The userlogins file is not enabled by default, while the errors log is.  The userlogins file can be optionally enabled (with CentOS and from what I can tell with Debian as well).

I can explain this in more detail if needed, but my aim was to make default fail2ban work with default roundcube - today it does not (regardless of platform). There are a lot of How-To's out there for getting fail2ban and roundcube to work, I just wanted to finally correct fail2ban.